### PR TITLE
modified intro, motivation and requirements

### PR DESCRIPTION
**Pull request by csilva-cl**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -1,38 +1,37 @@
...
+New technologies and developments are been discovered at an outstanding pace. Virtualization became a standard for the industry many years ago, but it has been replaced by new technologies, like micro-services and containerize applications, however, there are still some services that need to be run in virtual machine.
 
-Many of the services that are going to be mentioned as crucial to run in a Virtual Machine, can in fact capable of been deployed in a container, but there are security matters and loops avoidance that are important to prevent and consider:
+The following are some of the critical services for Rubin Observatory that are required to run in virtual machines:
 
-  - Rubin Kubernetes clusters run over Fixed DHCP IP, meaning that the members of the cluster are set to use DHCP, and on the other side, the DHCP Server holds a DHCP reservation for the server's MAC Address, so it always will get the same one.
```
Comment (hreinking) [inline index.rst:16]: I still believe is important to numerate the services that needs to run over VMs

File: index.rst [hunk 2]
```diff
@@ -1,38 +1,37 @@
...
-  - Same reasoning applies for DNS Servers; if the service doesn't exists until the container is created, it will create a conflic at kube-dns.
```
Comment (hreinking) [inline index.rst:17]: Also saying that even though they can be dockerize, we would end up in an "end and the chicken" scenario.

File: index.rst (hunk 1 above)
Comment (csilva-cl) [reply index.rst:16]: this is a feature of k8s not a requirement of virtualization

File: index.rst (hunk 2 above)
Comment (csilva-cl) [reply index.rst:17]: it is not a requirement to run dns on vms

Comment (hreinking) [reply index.rst:17]: My point is that it should: k8s needs a working DNS as well as the servers in the cluster.

File: index.rst (hunk 1 above)
Comment (hreinking) [reply index.rst:16]: the kubernetes platform is build over a cluster of servers, which needs an already running DHCP and DNS server. Also, to provision the servers, we would need a TFTP for PXE.

File: index.rst (hunk 2 above)
Comment (csilva-cl) [reply index.rst:17]: I understand, but it is not a requirement to run dns in vms

**Review by hreinking — APPROVED** [review]

**Pull request merged**